python-stdlib/time/time.py

```diff
@@ -0,0 +1,37 @@
+import re
+
+_TS_YEAR=0
+_TS_MON=1
+_TS_MDAY=2
+_TS_HOUR=3
+_TS_MIN=4
+_TS_SEC=5
+_TS_WDAY=6
+_TS_YDAY=7
+_TS_ISDST=8
+
```

Comment:
These probably should all be `const()`

In the latest mpy the lists below could be too, though I'm not sure how important back compatibility is here.